bedrock/firefox/views.py

```diff
@@ -368,8 +395,14 @@ class WhatsnewView(L10nTemplateView):
     ftl_files_map = {
         "firefox/developer/whatsnew.html": ["firefox/developer"],
         "firefox/developer/whatsnew-mdnplus.html": ["firefox/whatsnew/whatsnew-developer-mdnplus"],
-        "firefox/nightly/whatsnew.html": ["firefox/nightly/whatsnew", "firefox/whatsnew/whatsnew"],
-        "firefox/whatsnew/index.html": ["firefox/whatsnew/whatsnew-s2d", "firefox/whatsnew/whatsnew"],
+        "firefox/nightly/whatsnew.html": [
+            "firefox/nightly/whatsnew",
+            "firefox/whatsnew/whatsnew",
+        ],
+        "firefox/whatsnew/index.html": [
+            "firefox/whatsnew/whatsnew-s2d",
+            "firefox/whatsnew/whatsnew",
+        ],
         "firefox/whatsnew/index-thanks.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx125-na.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx125-eu.html": ["firefox/whatsnew/whatsnew"],
```

Comment:
also here is seems unrelated to this PR? the weird thing is not the whole file is reformatted, only some lines(?)